blocks/editable/index.js

```diff
@@ -36,41 +35,28 @@ import { EVENTS } from './constants';
 
 const { BACKSPACE, DELETE, ENTER } = keycodes;
 
-function createTinyMCEElement( type, props, ...children ) {
-	if ( props[ 'data-mce-bogus' ] === 'all' ) {
-		return null;
-	}
-
-	if ( props.hasOwnProperty( 'data-mce-bogus' ) ) {
-		return children;
-	}
-
-	return createElement(
-		type,
-		omitBy( props, ( value, key ) => key.indexOf( 'data-mce-' ) === 0 ),
-		...children
-	);
-}
-
 function isLinkBoundary( fragment ) {
 	return fragment.childNodes && fragment.childNodes.length === 1 &&
 		fragment.childNodes[ 0 ].nodeName === 'A' && fragment.childNodes[ 0 ].text.length === 1 &&
 		fragment.childNodes[ 0 ].text[ 0 ] === '\uFEFF';
 }
 
+function nodeListToString( nodeList ) {
+	const container = document.createElement( 'div' );
+	nodeList.forEach( ( node ) => container.appendChild( node ) );
+	return container.innerHTML;
```

Comment:
_Wonders if `Array.from( beforeFragment.childNodes ).map( node => node.innerHTML ).join( '' )` could work._

Reply:
I guess so, with `outerHTML` thought. but i think the text nodes don't have the `outerHTML` property, maybe better keeping this way for consistency. I don't know.

Reply:
Yep, you're right :)

Reply:
Fragments... 🙄